### ISSUE TITLE
About CCR-CLIP pretrained

### ISSUE THREAD

**Issue by ra890927** [FudanVI/FudanOCR#85]
Hello,
I download pretrained CCR-CLIP from baidu, and I load it and test on ICDAR 2013. I get accuracy about 50%, is far from paper zero-shot setting accuracy (72.98%). I don't know whether my validation code was wrong or pretrained model is not the best version. Please give me some suggestion or better pretrained CCR-CLIP model weight. Thanks.

**Comment by hyangyu** [comment]
Thank you for your attention!  The accuracy of zero-shot (72.98%) is obtained by training the CCR-CLIP model on corresponding **2755-CZS** datasets, which is defined in [Zero-shot Chinese Character Recognition with Stroke-Level Decomposition]. The uploaded pre-trained model is prepared for fine-tuning in the CTR settings.
